src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -194,9 +195,13 @@ class WorkspaceReimburseView extends React.Component {
                         <Text>{this.props.translate('workspace.reimburse.trackDistanceCopy')}</Text>
                     </View>
                     <OfflineWithFeedback
-                        errors={lodashGet(this.props, ['policy', 'customUnits', this.state.unitID, 'errors'])}
-                        pendingAction={lodashGet(this.props, ['policy', 'customUnits', this.state.unitID, 'pendingAction'])}
-                        onClose={() => Policy.removeUnitError(this.props.policyID, this.state.unitID)}
+                        errors={{
+                            ...lodashGet(this.props, ['policy', 'customUnits', this.state.unitID, 'errors'], {}),
+                            ...lodashGet(this.props, ['policy', 'customUnits', this.state.unitID, 'rates', this.state.unitRateID, 'errors'], {}),
+                        }}
+                        pendingAction={lodashGet(this.props, ['policy', 'customUnits', this.state.unitID, 'pendingAction'], '')
+                            || lodashGet(this.props, ['policy', 'customUnits', this.state.unitID, 'rates', this.state.unitRateID, 'pendingAction'], '')}
```

Comment:
This is producing a prop types error:
<img width="993" alt="Screen Shot 2022-08-30 at 4 01 34 PM" src="https://user-images.githubusercontent.com/7277067/187457422-43a2e7d8-7066-47de-b715-d154c588ab34.png">